spec.md

```diff
@@ -936,6 +976,8 @@ message NodePublishVolumeResponse {
 
 A Node Plugin MUST implement this RPC call.
 This RPC is a reverse operation of `NodePublishVolume`.
+This RPC MUST remove any mounts setup by the corresponding  `NodePublishVolume`.
+This Plugin SHALL assume that this RPC will be executed at least once for each successful `NodePublishVolume` call.
 If the corresponding Controller Plugin has `PUBLISH_UNPUBLISH_VOLUME` controller capability, the CO MUST guarantee that this RPC is called before `ControllerUnpublishVolume` is called for the given node and the given volume.
```

Comment:
I think we should call out that `ControllerUnpublishVolume` will be called by the CO after `NodeUnpublishVolume` is called for each `target_path` and is successful.